templates/trusted-node/cdk-node-config.toml

```diff
@@ -31,13 +31,9 @@ MaxPendingTx = 1
 {{if eq .zkevm_rollup_fork_id "12"}}
 MaxBatchesForL1 = 300
 BlockFinality="FinalizedBlock"
+{{end}}
 RPCURL = "http://{{.l2_rpc_name}}{{.deployment_suffix}}:{{.zkevm_rpc_http_port}}"
 GetBatchWaitInterval = "10s"
-{{end}}
-# FIXME - this shouldn't depend on a direct connection to the sequencer, but it seems to be failing if there are disconnects between the DS and RPC
-SanityCheckRPCURL = "http://{{.sequencer_name}}{{.deployment_suffix}}:{{.zkevm_rpc_http_port}}"
```

Comment:
Is this going to reintroduce warnings into the logs about not having a sanity check url enabled?

Reply:
No. The sanity check does not exist any more. Now the sequence sender only reads the RPC so it does not need to compare against the data stream.